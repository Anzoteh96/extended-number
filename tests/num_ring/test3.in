475874089
-443338684
6116444
