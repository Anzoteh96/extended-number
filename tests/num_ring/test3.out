Enter the number base b:
Enter two numbers n and m:
b is 475874089
n mod b is 32535405
m mod b is 6116444
n + m is 38651849
n - m is 26418961
n * m is 432035889
n ** m is 58731250
